0.15 Release Notes: VBAO (#1821)
bevyengine/bevy-website: release-content/0.15/release-notes/13454_vbao.md

> **Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://arxiv.org/abs/2301.11376) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique:

Reviewer: ```suggestion
**Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://arxiv.org/abs/2301.11376) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique and is particularly important on thin geometry (like the chair legs in the scene below):
```

> **Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://cybereality.com/screen-space-indirect-lighting-with-visibility-bitmask-improvement-to-gtao-ssao-real-time-ambient-occlusion-algorithm-glsl-shader-implementation/) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique:

Reviewer 2: I think we should've linked to the actual paper https://arxiv.org/abs/2301.11376 here instead

Reviewer: ```suggestion
**Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://arxiv.org/abs/2301.11376) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique:
```

> **Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://cybereality.com/screen-space-indirect-lighting-with-visibility-bitmask-improvement-to-gtao-ssao-real-time-ambient-occlusion-algorithm-glsl-shader-implementation/) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique:

Reviewer 2: The notable selling point of VBAO is that it works a lot better on thin geometry, like the chair legs. In other scenes it will have less of an impact.

Reviewer: ```suggestion
**Bevy 0.15** introduces a new Screen Space Ambient Occlusion (SSAO) algorithm: [Visibility Bitmask Ambient Occlusion](https://cybereality.com/screen-space-indirect-lighting-with-visibility-bitmask-improvement-to-gtao-ssao-real-time-ambient-occlusion-algorithm-glsl-shader-implementation/) (VBAO). VBAO builds on GTAO by adding a bitmask that allows multiple "sectors" of a considered half circle to be occluded, instead of just one angle. This improves the accuracy of the technique, and is particularly important on thin geometry (like the chair legs in the scene below):
```
